schema.graphql

```diff
@@ -9,11 +9,12 @@ enum ControlledTokenType {
   Sponsorship
 }
 
-enum YieldServiceTokenType {
-  cToken
+enum PrizePoolType {
+  Compound
+  yVault
+  Aave
```

Comment:
Let's leave out Aave for now. I can see yVault making since since it's close!

Reply:
done